.circleci/config.yml

```
@@ -61,6 +61,23 @@ jobs:
           paths:
             - ".git"
 
+  tidy-go:
+    executor: golang
+    steps:
+      - checkout
+      - run:
+          name: Check go mod tidy
+          # Use --check or --exit-code when available (Go 1.22?)
```

This is implemented now

```suggestion
          # Use --diff when available (Go 1.23)
```